edxval/tests/test_api.py

```diff
@@ -75,12 +126,12 @@ def test_create_profile(self):
         """
         Tests the creation of a profile
         """
-        result = api.create_profile(constants.PROFILE_DESKTOP)
+        api.create_profile(constants.PROFILE_DESKTOP)
         profiles = list(Profile.objects.all())
         self.assertEqual(len(profiles), 6)
-        self.assertEqual(
-            profiles[-1].profile_name,
-            constants.PROFILE_DESKTOP
+        self.assertIn(
+            constants.PROFILE_DESKTOP,
+            [unicode(profile) for profile in profiles],
```

Comment:
Note to reviewers: I had to update this test since it failed locally for me and would be flaky depending on the list of videos returned by the database.